app.js

```diff
@@ -46,7 +48,9 @@ app.use('/v1/analytics', analyticsRouter);
 
 app.use('/v2/analytics', v2AnalyticsRouter);
 app.use('/v2/stats', v2StatsRouter);
-app.use('/v3/stats/bno', v3StatsRouter);
+app.use('/v3/stats', v3StatsRouter);
+app.use('/v3/stats/bno', v3StatsBnoRouter);
+app.use('/v3/stats/bno', v3StatsWorldometerRouter);
```

Comment:
@gibtang should be `'/v3/stats/worldometer'`?

Reply:
Fixed typo in 9b76160505f729e76aa12c9a4fccd57a501a8c42